poppy/provider/akamai/services.py

```diff
@@ -1085,6 +1085,12 @@ def _get_configuration_number(self, domain_obj):
         return configuration_number
 
     def _get_provider_access_url(self, domain_obj, dp, edge_host_name=None):
+        """
+        :param domain_obj: domain object
+        :param dp:
```

Comment:
Finish docstring.  (See `HACKING.rst` for formatting details) 

Include type info . (*tip*: pycharm debugger can be used to aid with types https://www.jetbrains.com/help/pycharm/using-docstrings-to-specify-types.html)